redaxo/src/core/lib/rex.php

```diff
@@ -316,6 +316,31 @@ public static function getConsole()
         return self::getProperty('console', null);
     }
 
+    /**
+     * @param int $db
+     * @psalm-param positive-int $db
+     *
+     * @throws rex_exception
+     *
+     * @return rex_config_db
+     */
+    public static function getDbConfig($db = 1)
+    {
+        $configFile = rex_path::coreData('config.yml');
+
+        if (!$configFile) {
+            throw new rex_exception('Unable to read "'. $configFile .'"');
+        }
+
+        $config = rex_file::getConfig($configFile);
```

Comment:
yaml parsing ist recht teuer, deswegen cachen wir ja die config.yml.
Und die db config sollte hier ja in `self::getProperty` bereits enthalten sein.

Reply:
Die stellen die ich bisher ersetzt hab, haben die datei auch erst gelesen und geparsed.

Ich denke hier könnte man jetzt mittels getProperty() arbeiten.. muss mal testen ob das für alle call-sites funktionieren würde